src/js/modules/infragistics.ui.editors.js

```diff
@@ -5758,7 +5755,7 @@
 
 							// Raise Warning level 2
 							this._sendNotification("warning",
-								$.ig.util.stringFormat($.ig.Editor.locale.minValExceedSetErrMsg,
+								$.ig.util.stringFormat(this._getLocaleValue("maxValExceedSetErrMsg"),
```

Comment:
Please, check if you didn't change minValExceedSetErrMsg with maxValExceedSetErrMsg